src/redux/Restaurant/selectors.js

```diff
@@ -244,3 +244,8 @@ export const selectIsActionable = createSelector(
     );
   },
 );
+
+export const selectIsPrinting = createSelector(
```

Comment:
There is a selector called `selectPrintingOrderId`. It can be used either here or directly from a React component